src/messages/iscn.ts

```diff
@@ -48,14 +48,15 @@ export function formatISCNPayload(payload: ISCNSignPayload, version = 1): {
 }
 
 export function formatMsgCreateIscnRecord(
-  senderAddress: string, payload: ISCNSignPayload,
+  senderAddress: string, payload: ISCNSignPayload, nonce = 0,
```

Comment:
Aren't nonce optional in chain msg format? Can we not have a default 0? Would that prevent the test case diff below?

Reply:
Yes, it seems better not to assign `nonce` here.